### ISSUE TITLE
[BITV] 9.1.3.1a/9.1 - The page lacks an "h1" heading, and the existing "h2" heading, "Groups," does not accurately describe the content of the page and therefore cannot serve as the main heading for the page. (1)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41829]
It is recommended to use the text "Active users" for the "h1" heading.

https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-be75bbae42-v9-n1

**Comment by JuliaKirschenheuter** [comment]
![Screenshot from 2023-12-14 17-34-46](https://github.com/nextcloud/server/assets/6078378/3a9cc04f-cfa2-4cfe-b459-87613145122d)

Groups is not `<h2>`. 

`<h1 class="hidden-visually"> Active users </h1>` is already there.

@Pytal is this issue resolved (because it remains in progress)?

**Comment by Pytal** [comment]
> Groups is not `<h2>`.

You mean *is*?

> @Pytal is this issue resolved (because it remains in progress)?

Yes :)
